exercise3/calculateFactorial.js

```diff
@@ -0,0 +1,10 @@
+function calculateFactorial(number) {
+  if (number === 0 || number === 1) return 1;
+
+  let factorial = 1;
+  for (let index = 2; index <= number; index++) {
+    factorial *= index;
+  }
+
+  return factorial;
+}
```

Comment:
Great job 👍. `SUGGESTIONS:`  Do you mind if you refactor this code using `recursive function/ method` for more information read[ recursive functions](https://www.javascripttutorial.net/javascript-recursive-function/). This will enable you to solve this challenge in just one one of code if you combine it with `ternary operators`

Reply:
Thank you for the comments but I found that recursive functions can be hard to understand
may be you can elaborate more on why I need to use recursive function